modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -535,4 +584,8 @@ If new Infinite Scale releases are available, you *must not* skip any version in
 * If there is no release gap, you can update by just stopping and starting the compose environment.
 * For *any* https://owncloud.dev/ocis/release_roadmap/#dates[release gap], you must stop the compose environment with `docker compose down`, set the `OCIS_DOCKER_TAG` variable in the `.env` file accordingly, pull the new image with `docker compose pull` and start the compose environment with `docker compose up -d` again. For the last release, you can remove any setting of `OCIS_DOCKER_TAG` as `latest` is used by default.
 
+== Certificate Renewal
+
+There is nothing special to do renewing certificates issued. Traefik takes care on the renewal process automatically. In case of a renewal issue, just restart the compose environment as traefik checks certificate expiry on startup.
```

Comment:
```suggestion
There is nothing special to do renewing certificates issued. Traefik takes care of the renewal process automatically. In case of a renewal issue, just restart the compose environment as Traefik checks certificate expiry on startup.
```